packages/augur-ui/src/modules/market/components/common/comments/market-comments.tsx

```diff
@@ -1,160 +1,103 @@
+      await threeBoxInstance.syncDone;
 
-      const space = await box.openSpace('augurtesteightportis', {});
+      const space = await threeBoxInstance.openSpace('augur', {});
       await space.syncDone;
+
+      publicProfile = await Box.getProfile(addressFromProvider);
     } catch (error) {
       console.error(error);
-      // return
+      return;
     }
 
-    this.setState({box, address, isReady: true, provider});
+    setBox(threeBoxInstance);
+    setAddress(addressFromProvider);
+    setProfile(publicProfile);
+    setIsReady(true);
   };
 
-  render () {
-    const {
-      box,
-      address,
-      isReady,
-      provider
-    } = this.state;
-
-    return (
-      <section className={Styles.Comments}>
-        {isReady && (
-          <Comments
-            // required
-            spaceName="augurtesteightportis"
-            threadName={this.props.marketId}
-            adminEthAddr={address}
-
-            // Required props for context A) & B)
-            box={box}
-            currentUserAddr={address}
-
-            // Required prop for context B
-            // loginFunction={this.handleLogin}
-
-            // Required prop for context C)
-            // ethereum={provider._web3Provider}
-
-            // optional
-            // members={false}
-            // showCommentCount={10}
-            // threadOpts={{}}
-            // useHovers={false}
-            // currentUser3BoxProfile={currentUser3BoxProfile}
-            // userProfileURL={address => `https://mywebsite.com/user/${address}`}
-          />
-        )}
-      </section>
-    );
-  }
-}
-
-// import React from 'react';
-// import { FacebookComments } from 'modules/market/components/common/comments/facebook-comments';
-//
-// interface MarketCommentsProps {
-//   marketId: string;
-//   colorScheme: string;
-//   numPosts: number;
-//   networkId: string;
-//   whichCommentPlugin?: string;
-// }
-//
-// export const MarketComments = ({
-//   marketId,
-//   colorScheme,
-//   numPosts,
-//   networkId,
-//   whichCommentPlugin,
-// }: MarketCommentsProps) => {
-//   return (
-//     whichCommentPlugin === 'facebook' && (
-//       <FacebookComments
-//         marketId={marketId}
-//         colorScheme={colorScheme}
-//         numPosts={numPosts}
-//         networkId={networkId}
-//       />
-//     )
-//   );
-// };
+  return (
+    <section className={Styles.Comments}>
+      {whichCommentPlugin === '3box' && isReady && (
+        <ThreeBoxComments
+          // required
+          spaceName="augur"
+          threadName={marketId}
+          adminEthAddr={adminEthAddr}
+
+          // Required props for context A) & B)
+          box={box}
+          currentUserAddr={address}
+
+          // optional
+          showCommentCount={numPosts}
+          currentUser3BoxProfile={profile}
+          // useHovers={true}
+        />
+      )}
+      {whichCommentPlugin === 'facebook' && (
```

Comment:
Should we just drop facebook now?

Reply:
Paul asked me to leave it as a choice. The effort for styling and all is already done and it adds very little to the final bundle, I guess